dockerfiles/centos_7

```diff
@@ -108,7 +108,6 @@ gpgcheck=0\n" > /etc/yum.repos.d/tarantool_2.repo \
         postgresql-devel \
         cyrus-sasl-devel \
         libev-devel \
-        wget \
```

Comment:
Shouldn't we add `curl` here instead of removing `wget`?

Reply:
Curl already present in CentOS 7 docker image.